.storybook/preview.tsx

```diff
@@ -106,6 +106,15 @@ export const globalTypes: GlobalTypes = {
       title: "Scaffold",
     },
   },
+  styleInjection: {
```

Comment:
Is this toggle here for us to see components without css applied? Given in theory, QA stories can handle different situation without global setting.

Reply:
Yes...found it useful when testing things so left it in